package.json

```
@@ -5,15 +5,26 @@
   "scripts": {
     "postinstall": "gulp",
     "start": "node --max-http-header-size=80000 ./bin/www",
-    "prepare": "npx snyk protect || npx snyk protect -d || true"
+		"prepare": "npx snyk protect || npx snyk protect -d || true"
```

looks like this got changed from spaces to tabs
```suggestion
    "prepare": "npx snyk protect || npx snyk protect -d || true"
```